clang/test/dpct/cub/blocklevel/blockload.cu

```diff
@@ -0,0 +1,83 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct -in-root %S -out-root %T/blocklevel/blockload %S/blockload.cu --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/blocklevel/blockscan/blockload.dp.cpp --match-full-lines %s
+
+#include <iostream>
+#include <vector>
+
+#include <cuda_runtime.h>
+#include <cub/cub.cuh>
+
+#define WARP_SIZE 32
+
+const int N = 256;
+const int BlockSize = 128;
+const int ItemsPerThread = 4;
+
+
+void init_data(int* data, int num) {
+  for(int i = 0; i < num; i++)
+    data[i] = i;
+}
+void verify_data(int* data, int num) {
+  return;
+}
+void print_data(int* data, int num) {
+  for (int i = 0; i < num; i++) {
+    std::cout << data[i] << " ";
+  }
+  std::cout << std::endl;
+}
+
+//CHECK: void SumKernel(int* data,

```

Comment:
This lit need update, usually, we check migration result here, and make sure the result is correct.

Reply:
Do we need to add a sycl kernel for blockload here? for reduce /exclusive /inclusive scans we have sycl::reduce_over_group etc apis . Do we need such api specifically for load as well?